### PR TITLE
Update node authors script.

### PR DESCRIPTION
**Pull request by angaither**
Refs #1417
will fix, once run in production.

### CODE REVIEW

**Comment by aaronschachter** [comment]
Looks ok. Kinda OCD, but could be a relatively small lift to update the node_revisions table as well... but in this case we'd need to check more than just the original author, as I know Ben has made revisions.

**Comment by angaither** [comment]
@aaronschachter updated!

File: scripts/update-node-authors.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  9  => 1193105,  /*'nmody@dosomething.org',         */
+  11 => 1192054,  /*'aruderman@dosomething.org',     */
+  14 => 1157702,  /*'jlorch@dosomething.org',        */
+  18 => 1258186,  /*'dfurnes@dosomething.org',       */
+  19 => 547446,   /*'bclark@dosomething.org',        */
+  22 => 329195,   /*'mfantini@dosomething.org',      */
+);
+
+// Need to update the revisions first, otherwise it zeros out uids
+$revision_results = db_query("SELECT nid, vid, uid from node_revision");
```
Comment (aaronschachter) [inline scripts/update-node-authors.php:33]: Aren't there some authors that would be returned in this list that don't appear in the $authors array?  I'm mostly thinking of Ben, hasn't he made revisions?  But would good to be sure if there are any other missing uid's as well...

Comment (angaither) [inline scripts/update-node-authors.php:33]: Did he make revisions on stage or production? When I run a query getting all the users his email isn't returned.

`SELECT DISTINCT u.uid, u.mail FROM users u INNER JOIN node  n on n.uid = u.uid INNER JOIN node_revision nr on nr.uid = u.uid;`

Comment (angaither) [inline scripts/update-node-authors.php:33]: Nevermind, my db was jenky after I ran the update. duhhh

**Comment by angaither** [comment]
@aaronschachter for real this time.

File: scripts/update-node-authors.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+foreach($revision_results as $result) {
+  $old_uid = $result->uid;
+  $new_uid = $authors[$old_uid];
+  if ($new_uid)  {
+    echo "Updating node revision: " . $result->nid . ' : ' . $result->vid . " from " . $old_uid . " to " . $new_uid . "\n";
+    db_update('node_revision') // Table name no longer needs {}
```
Comment (aaronschachter) [inline scripts/update-node-authors.php]: Doesn't this guy need a WHERE condition?

**Comment by aaronschachter** [comment]
Looks good :+1:

**Pull request merged**